keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -169,6 +194,151 @@ import (
 )
 ```
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: DisableCloudProviders
+    - Components depending on the feature gate: kubelet, kube-apiserver, kube-controller-manager
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: DisableKubeletCloudCredentialProvider
+    - Components depending on the feature gate: kubelet
+
+* **Does enabling the feature change any default behavior?**
+  Yes, enabling this feature will disable all capabilities enabled when --cloud-provider is set.
+  Users need to ensure they have migrated to out-of-tree components prior to enabling this feature gate.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes, the feature can be disabled once it is enabled. If disabled, users must ensure
+  any out-of-tree counterpart components are no longer running in the cluster.
```

Comment:
I believe both the Credential Provider and APIServer Network Proxy are safe. Its only the CCM which we need to ensure is no longer running.

Reply:
updated